homeassistant/components/onvif/event.py

```diff
@@ -82,11 +82,13 @@ async def async_start(self) -> bool:
         if await self.device.create_pullpoint_subscription():
             # Initialize events
             pullpoint = self.device.create_pullpoint_service()
-            await pullpoint.SetSynchronizationPoint()
-            req = pullpoint.create_type("PullMessages")
-            req.MessageLimit = 100
-            req.Timeout = dt.timedelta(seconds=5)
-            response = await pullpoint.PullMessages(req)
+            # await pullpoint.SetSynchronizationPoint()
```

Comment:
Git has a history, so no need to comment out code. Just replace it with the new logic.